apps/contrib/views.py

```diff
@@ -11,5 +11,5 @@ class SearchResultsView(ListView):
     def get_queryset(self):
         sb = get_search_backend()
         query = self.request.GET.get('q')
-        res = sb.search(query, Page, fields=get_search_fields())
+        res = sb.autocomplete(query, Page, fields=get_search_fields())
```

Comment:
The search changes will need some manual testing, I haven't done that yet

Reply:
I did a minimal testing locally LGTM